crates/primitives/starknet/src/transaction/types.rs

```diff
@@ -372,6 +387,114 @@ impl From<DeployAccountTransaction> for Transaction {
         }
     }
 }
+
+/// Error of conversion between the Madara Primitive Transaction and the RPC Transaction
+#[cfg(feature = "std")]
+#[derive(Debug)]
+pub enum RPCTransactionConversionError {
+    /// The u8 stored version doesn't match any of the existing version at the RPC level
+    UnknownVersion,
+    /// Missing information
+    MissingInformation,
+    /// Conversion from byte array has failed.
+    FromArrayError,
+    /// Provided byte array has incorrect lengths.
+    InvalidLength,
```

Comment:
I think we can just put an RPC error tulle here, instead of redefining the same errors

Reply:
noob question, not sure what you mean